.github/workflows/jdk-early-access-build.yml

```diff
@@ -70,27 +70,26 @@ jobs:
       - name: Build with Maven
         # -fae to try to gather as many failures as possible
         # (but not maven.test.failure.ignore because test report generation is buggy)
-        # Gradle bits are excluded due to https://github.com/gradle/gradle/issues/13481
         # descriptor-tests (integration-tests/maven) are disabled due to https://github.com/quarkusio/quarkus/issues/16862 (and/or 16806)
         run: |
-          ./mvnw $JVM_TEST_MAVEN_OPTS -Dtcks install -fae -pl '!devtools/gradle' -pl '!integration-tests/gradle' -Dno-descriptor-tests
+          ./mvnw $JVM_TEST_MAVEN_OPTS -Dtcks install -fae -Dno-descriptor-tests
       # Test reports disabled due to: https://github.com/ScaCap/action-surefire-report/issues/39
```

Comment:
Doesn't look like this issue was fixed so better leave those `action-surefire-report` steps disabled. Is this a leftover from a run in your fork?